.travis.yml

```diff
@@ -0,0 +1,36 @@
+branches:
+  only:
+    - master
+
+jobs:
+  include:
+    - stage: test
+      language: node_js
+      node_js:
+        - 8
+      cache: yarn
+      script:
```

Comment:
Remove Yarn, because you don't have any tests!

Reply:
no